backend/EasyMeets.Core/EasyMeets.Core.WebAPI/Controllers/AvailabilityController.cs

```diff
@@ -1,12 +1,12 @@
 ﻿using EasyMeets.Core.BLL.Interfaces;
 using EasyMeets.Core.Common.DTO.Availability;
+using EasyMeets.Core.Common.DTO.Availability.NewAvailability;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 
 namespace EasyMeets.Core.WebAPI.Controllers
 {
-    [Authorize]
```

Comment:
Why removed authorize?

Reply:
To test the endpoint with swagger, pushed it accidentally, sorry